python/common/org/python/types/Bool.java

```diff
@@ -91,9 +100,9 @@ public Bool(org.python.Object[] args, java.util.Map<java.lang.String, org.python
     )
     public org.python.Object __lt__(org.python.Object other) {
         if (other instanceof org.python.types.Int) {
-            return new org.python.types.Bool((((org.python.types.Bool) this).value ? 1 : 0) < ((org.python.types.Int) other).value);
+            return org.python.types.Bool.getBool((((org.python.types.Bool) this).value ? 1 : 0) < ((org.python.types.Int) other).value);
         } else if (other instanceof org.python.types.Bool) {
-            return new org.python.types.Bool((((org.python.types.Bool) this).value ? 1 : 0) < (((org.python.types.Bool) other).value ? 1 : 0));
+            return org.python.types.Bool.getBool((((org.python.types.Bool) this).value ? 1 : 0) < (((org.python.types.Bool) other).value ? 1 : 0));
```

Comment:
Another tweak, but this expression can be simplified as well; if we're comparing two bools, there's no need to do a transition through integer logic to get back to a boolean.

Reply:
I'm not sure we can use `<` directly on booleans here as that's not defined in Java, but other places in the file like `==` or `|` have been tweaked!